src/Components/Footer/Footer.js

```diff
@@ -1,7 +1,249 @@
 import React from 'react';
+import Category from './Category/Category';
+import ContentAsset from './ContentAsset/ContentAsset';
+import ContentCards from './ContentCards/ContentCards';
+import ContentInfo from './ContentInfo/ContentInfo';
+import Copyright from './ContentCopyright/ContentCopyright';
+import './Footer.scss';
 
 const Footer = () => {
-  return <div>footer</div>;
+  return (
+    <footer className="footer">
+      <div className="footerTitle">
+        <div className="footerText">
+          <p className="headTxt">New Kidsney® Visa® Cardmembers can earn a</p>
+          <h3 className="midTxt">
+            <span className="currency">$</span>
+            <span className="amount">100</span>
+            <span className="txt">Statement Credit</span>
+          </h3>
+          <div className="links">
+            <span className="linksTxt">Restrictions apply.</span>
+            <button class="linksBtn">Learn More</button>
+          </div>
+        </div>
+        <div className="footerImg">
+          <img
+            src="https://images.unsplash.com/photo-1572635148619-a66d0ddbc9bc?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80"
+            alt="cards"
+          />
+        </div>
+      </div>
+      <div className="footerContainer">
+        <div className="signUp">
+          <div className="signUpTxt">
+            Sign up for our emails to get the inside scoop on special offers and
+            new products.
+          </div>
+          <div className="signUpLinks">
+            <button className="signUpBtn">Sign up</button>
+            <div className="privacyPolicy">
+              <span className="message">Stay in Touch</span>
+              <span className="policy">Privacy Policy</span>
+            </div>
+          </div>
+          <div className="socialIcons">
+            <i class="fab fa-facebook-f" />
+            <i class="fab fa-instagram" />
+            <i class="fab fa-twitter" />
+            <i class="fab fa-pinterest-p" />
+          </div>
+        </div>
+        <div className="items">
+          <div className="category">
+            <ul className="categoryList">
+              {FOOTER_CATEGORY_LIST.map(category => {
+                return (
+                  <Category key={category.id} content={category.content} />
+                );
+              })}
```

Comment:
화살표 함수에서, 다른 로직 없이 단순 return만 하는 함수라면 다음처럼 중괄호와 return문 없이 소괄호로만 사용할 수 있습니다.
```suggestion
              {FOOTER_CATEGORY_LIST.map(category => (
                <Category key={category.id} content={category.content} />
              ))}
```
표현만 다를 뿐 위의 두 함수는 같습니다!